Zscaler plugin - urls and domains ad input parameter  in actions Blacklist URL and Lookup URL (#693)
rapid7/insightconnect-plugins: zscaler/help.md

# Version History

> * 1.1.2 - Change input `URLs` in actions Blacklist URL and Lookup URL to accepts URls and domains

Reviewer: ```suggestion
* 1.1.2 - Support both domains and URLs in the Blacklist URL and Lookup URL actions
```
